test/unit/web/framework/test_framework_base.py

```diff
@@ -0,0 +1,18 @@
+from galaxy.web.framework.base import WebApplication
```

Comment:
Should this file path be `test/unit/web_framework/test_framework_base.py` instead?